src/v2/guide/single-file-components.md

```diff
@@ -1,48 +1,48 @@
 ---
-title: Single File Components (Componentes de un solo archivo)
+title: Componentes de un Solo Archivo (Single File Components)
 type: guide
 order: 402
 ---
 
 ## Introducción
 
-En muchos proyectos de Vue, los componentes globales serán definidos usando `Vue.component`, seguido de `new Vue({ el: '#container' })` para apuntar a un elemento contenedor en el body de cada página.
+En muchos proyectos de Vue los componentes globales son definidos usando `Vue.component`, seguido de `new Vue({ el: '#container' })` para apuntar a un elemento contenedor en el cuerpo de cada página.
 
-Esto puede funcionar muy bien para proyectos de tamaño pequeño a mediano, donde JavaScript es usado solamente para mejorar algunas vistas. Sin embargo, en proyectos con mayor complejidad, or donde tu front-end es enteramente basado conducido por JavaScript, estas desventajas se hacen evidentes:
+Esto puede funcionar muy bien para proyectos pequeños y medianos, donde JavaScript solo se usa para mejorar ciertas vistas. Sin embargo, en proyectos más complejos, o cuando la interfaz está exclusivamente impulsada por JavaScript, varias desventajas se hacen evidentes:
 
-- **Definiciones globales** fuerza a nombres unicos para cada componente
-- **Plantilla de texto** carecen de resaltado de sintaxis y requieren el uso de barras para HTML multilínea
-- **Sin soporte CSS** significa que mientras el HTML y JavaScript son modularizados en componentes, el CSS es omitido
-- **Sin compilación** nos restringe a HTML y ES5 JavaScript, en lugar de preprocesadores como Pug (anteriormente Jade) y Babel
+- Las **definiciones globales** nos obligan a usar nombres únicos para cada componente
+- Las **plantillas de texto** carecen de resaltado de sintaxis y requieren feas barras para generar HTML multilínea
+- La **ausencia de soporte para CSS** significa que, en tanto HTML y JavaScript son modularizados en componentes, el CSS queda visiblemente fuera
+- **Sin una etapa de compilación** nos vemos restringidos a HTML y JavaScript ES5, en lugar de preprocesadores como Pug (anteriormente Jade) y Babel
 
-Todos estos puntos estan resueltos por **single-file components** con extensión `.vue`, lo cual es posible con herramientas como Webpack or Browserify.
+Todo esto se resuelve mediante componentes de un solo archivo con extensión .vue, hecho posible con herramientas de compilación como Webpack o Browserify.
```

Comment:
I preffer to use `lo cual es` over `hecho`, let's what other guys thinks about this :thinking:
cc: @sxcamacho @FLarra

Reply:
Done

Reply:
Lo cambié por "que pueden usarse con herramientas de compilación como Webpack o Browserify". No queda claro qué es "lo cual es hecho".